test/elementhandle.spec.ts

```diff
@@ -380,10 +379,9 @@ describe('ElementHandle specs', function () {
     });
     it('should wait correctly with waitFor', async () => {
       const { page, puppeteer } = getTestState();
-      puppeteer.__experimental_registerCustomQueryHandler(
-        'getByClass',
-        (element, selector) => element.querySelector(`.${selector}`)
-      );
+      puppeteer.__experimental_registerCustomQueryHandler('getByClass', {
+        queryOne: (element, selector) => element.querySelector(`.${selector}`),
+      });
```

Comment:
Let's add some tests for a handler that supports both `queryOne` and `queryAll` (which I suspect will be the common case)

Reply:
Good idea! I have added a couple in d83b8b3.